src/daft-sql/src/planner.rs

```diff
@@ -246,6 +246,17 @@ impl SQLPlanner {
                     .collect::<SQLPlannerResult<Vec<_>>>()?;
             }
         }
+        let sort_exprs = query
```

Comment:
```suggestion
        let sort_exprs = query
            .order_by
            .as_ref()
            .map(|order_by| {
                if order_by.interpolate.is_some() {
                    unsupported_sql_err!("ORDER BY [query] [INTERPOLATE]");
                }
                // TODO: if ordering by a column not in the projection, this will fail.
                self.plan_order_by_exprs(&order_by.exprs)
            })
            .transpose()?;
```

not sure how to make multiline suggestion btw but can avoid clone